<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);
color: white;
font: 10pt &quot;Segoe UI Historic&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox, QPushButton, QSpinBox, QDoubleSpinBox, QLineEdit {
	background: rgb(33, 33, 33);
    border-radius: 4px;
    padding-left: 7px;
	min-width: 150px;
	max-width: 150px;
	min-height: 25px;
	max-height: 25px;
}

QComboBox:hover, QPushButton:hover, QSpinBox:hover, QDoubleSpinBox:hover, QLineEdit:hover {	
	background-color: rgb(56, 56, 56);
}

QComboBox:pressed, QPushButton:pressed, QSpinBox:pressed, QDoubleSpinBox:pressed, QLineEdit:pressed {
	background-color: rgb(50, 50, 50);
}

QComboBox:drop-down, QSpinBox:down-button, QSpinBox:up-button, QDoubleSpinBox:down-button, QDoubleSpinBox:up-button {
    width: 10px;
	background-color: rgb(66, 66, 66);
}

QSpinBox:down-button, QSpinBox:up-button, QDoubleSpinBox:down-button, QDoubleSpinBox:up-button { height: 12px; }

QComboBox:drop-down:hover, QSpinBox:down-button:hover, QSpinBox:up-button:hover, QDoubleSpinBox:down-button:hover { background-color: rgb(72, 77, 82); }

QScrollBar:vertical
{
	background-color: #2A2929;
    width: 15px;
    margin: 15px 3px 15px 3px;
    border: 1px transparent #2A2929;
    border-radius: 4px;
}

QScrollBar::handle:vertical
{
	background-color: rgb(96, 95, 95);
    min-height: 5px;
    border-radius: 4px;
}

QScrollBar::handle:vertical:hover
{
	background-color: rgb(86, 85, 85);
    min-height: 5px;
    border-radius: 4px;
}

QScrollBar::sub-line:vertical, QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical:hover, QScrollBar::sub-line:vertical:on, QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on, QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on, QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical, QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
   	background: none;
}

 QTabWidget::pane { 
     position: absolute;
 }

QTabBar::tab:selected {
	background-color:rgb(45, 45, 45);
}

QTabBar::tab:!selected {
	background-color: rgb(33, 33, 33);
}
</string>
   </property>
   <layout class="QGridLayout" name="lay_main">
    <item row="0" column="1">
     <widget class="QFrame" name="frame_canvas">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 4px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="lay_canvas">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="maximumSize">
       <size>
        <width>244</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="main_tab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>400</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QScrollArea {
	border: none;
}</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>220</width>
             <height>637</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="lay_settings">
              <item row="0" column="0">
               <widget class="QWidget" name="wid_objects" native="true">
                <layout class="QGridLayout" name="gridLayout">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>6</number>
                 </property>
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <item row="1" column="0">
                  <widget class="QComboBox" name="cobox_objects">
                   <property name="cursor">
                    <cursorShape>PointingHandCursor</cursorShape>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="lbl_obj">
                   <property name="text">
                    <string>Object:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>177</width>
                <height>503</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="anim_tab">
       <attribute name="title">
        <string>Animation</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="wid_anim" native="true">
          <layout class="QGridLayout" name="lay_anim">
           <item row="1" column="0">
            <widget class="QComboBox" name="cobox_animParam">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QDoubleSpinBox" name="spbox_animSpeed">
             <property name="decimals">
              <number>3</number>
             </property>
             <property name="minimum">
              <double>-99.000000000000000</double>
             </property>
             <property name="maximum">
              <double>99.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.001000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Speed:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_animParam">
             <property name="text">
              <string>Parameter:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Length (sec):</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QSpinBox" name="spbox_length">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>443</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
